packages/skeleton-loader/src/SkeletonLoader.spec.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { render, within } from '@testing-library/react';
+import { axe } from 'jest-axe';
+
+import { CardSkeleton } from './CardSkeleton';
+import { FormSkeleton } from './FormSkeleton';
+import { ParagraphSkeleton } from './ParagraphSkeleton';
+import { Skeleton } from './Skeleton';
+import { TableSkeleton } from './TableSkeleton';
+
+describe('packages/skeleton-loader/Skeleton', () => {
```

Comment:
Would it be better to add these tests to each package instead of all to this one file?